src/main/java/gov/nasa/pds/label/Label.java

```diff
@@ -655,7 +655,7 @@ private DataObject makeTable(gov.nasa.arc.pds.xml.generated.File file, TableDeli
       } else if (table.getObjectLength() != null) {
         size = table.getObjectLength().getValue().longValueExact();
       } else if (file.getFileSize() != null) {
-        size = file.getFileSize().getValue().longValue() - offset;
+        size = 0;//file.getFileSize().getValue().longValue() - offset;
```

Comment:
@jordanpadams 

No. See detailed information on 781.

In 781 the table is N rows of 2 ascii_integer and 2 ascii_string. Hence, the smallest size is 8 bytes to include delimiter and 2**32 - 1 bytes because strings do have a fixed length. There are no hints to size other than offset of the next possible table.

Second, the code is processing THIS table. It has no idea nor, given the code structure, know if it is first, last, only, middle, or anything else. So, it cannot use offset from another table in the same file. Could do a big static map of file name and state information about the file -- way beyond reading/parsing an XML file -- that would introduce a huge host of processing problems like labels with same file that validate has not found yet because parsing is before validating etc.

This is a designed in failure. The best solution is to force object length in all tables, but that is not in our scope. It leaves us with a designed in failure where we just have to wait for validate or some later stage to have enough information to declare a problem.

Reply:
@jordanpadams 

No, my best hope is that it breaks some  other test that will give a hint to some other middle ground.